lib/cli.js

```diff
@@ -75,12 +75,24 @@ yargs
                     type: 'boolean',
                     describe:
                         'optional for asset-message: runs refresh command for related triggeredSends after deploy',
+                })
+                .option('execute', {
+                    type: 'boolean',
+                    describe: 'optional for query: runs execute after deploy',
                 });
         },
         handler: (argv) => {
             Mcdev.setOptions(argv);
 
-            Mcdev.deploy(argv.BU, csvToArray(argv.TYPE), csvToArray(argv.KEY), argv.fromRetrieve);
+            Mcdev.deploy(
+                argv.BU,
+                csvToArray(argv.TYPE),
+                csvToArray(argv.KEY),
+                argv.fromRetrieve
+                // ,
+                // argv.refresh,
+                // argv.execute
```

Comment:
Mcdev.setOptions() should take care of this already. That way, we can test for Util.OPTIONS.execute instead of having to use function parameters.

The parameter fromRetrieve was added earlier and should eventually be removed as well here and on lower methods